### PR TITLE
Reporter improvements

### LINKED ISSUE #1337: Should we move pass / fail count to the end of the output?
[Issuehunt badges](https://issuehunt.io/r/avajs/ava/issues/1337)

When there are errors, the pass/fail count shows at the top. This is almost always out of view once all the error messages are displayed.

I believe most other reporters show that as the final lines of the output. Should we follow suite?

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $60.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($60.00)

### Submitted pull Requests
- [#2487 Move fail / pass counter to the bottom of the test output](https://issuehunt.io/r/avajs/ava/pull/2487)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/avajs/ava/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
</details>

### PR DESCRIPTION
**Pull request by m5x5**
This PR should fix #1337.

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#1337: Should we move pass / fail count to the end of the output?](https://oss.issuehunt.io/repos/26820798/issues/1337)
- [#2046: feature request: let's add failed test count at the bottom if there are more than X failed](https://oss.issuehunt.io/repos/26820798/issues/2046)
---
</details>

### CODE REVIEW

**Comment by novemberborn** [comment]
Thanks for the PR @Michael55555! It's thrown me of course for today as I've spent several hours tweaking the reporter outputs 😱 I'm about to push the changes, let me know what you think.

Most of it is whitespace tweaks, though I've also updated the mini reporter to print the stats at the end. And I've added a subtle separator between the different output sections.

**Comment by Michael55555** [comment]
Looks good so far, thank you for your help! I was trying to install Ubuntu to test how it runs on a different OS, but I am glad you did it 😃.

**Review by m5x5 — COMMENTED** [review]
Are you going to add this code back, or do we not need it anymore? Because it was on top of the function.

**Comment by novemberborn** [comment]
> Are you going to add this code back, or do we not need it anymore? Because it was on top of the function.

Could you elaborate? GitHub isn't showing what this is in response to.

File: lib/reporters/verbose.js [hunk 1]
```diff
@@ -449,13 +449,6 @@ class VerboseReporter {
 			this.lineWriter.writeLine(colors.error(`${this.previousFailures} previous ${plur('failure', this.previousFailures)} in test files that were not rerun`));
 		}
 
-		if (this.stats.passedKnownFailingTests > 0) {
-			this.lineWriter.writeLine();
-			for (const evt of this.knownFailures) {
-				this.lineWriter.writeLine(colors.error(this.prefixTitle(evt.testFile, evt.title)));
-			}
-		}
-
```
Comment (Michael55555) [inline lib/reporters/verbose.js]: I mean this part

Comment (novemberborn) [reply lib/reporters/verbose.js]: Yea it seems weird to have it after the stats, and I don't know why we have to draw specific attention to it here.

Comment (Michael55555) [reply lib/reporters/verbose.js]: Ok, I see. It seems like everything is working as expected 🤔

**Pull request merged**